### ISSUE TITLE
Opensource License

### ISSUE THREAD

**Issue by CodePoetSarah** [MiSTer-devel/Main_MiSTer#110]
I am working on an article for opensource.com on the MiSTer project, we are blocked on using it though as much as the project is listed as opensource, there's no license information for the core project. Is this project opensource and under which license? 

https://opensource.org/licenses

Feel free to reach out if you have any questions :-) we have put it on hold until we can get clarification.

On a personal note, this is an awesome project.

**Comment by sorgelig** [comment]
Different cores may use different licenses. It's not a monolith single project.

**Comment by CodePoetSarah** [comment]
Thanks, it's the main MiSTer project that doesn't have a license. It's listed as opensource on the donate page, but there's no GPL in the code.

**Comment by CodePoetSarah** [comment]
You closed the ticket, does this mean the project will not be opensourced or it will? having clarification either way is good enough to close the ticket.

**Comment by sorgelig** [comment]
It's open sourced. Sources are opened, isn't?
I will think about license, but it has no relation to open source.

**Comment by CodePoetSarah** [comment]
Thanks, making the sources open doesn't make it opensource, it needs a license. There are a number which could apply, the most common is the GPL which a lot of the cores use. Let me know if I can help with anyway, we often work with developers when writing articles for opensource.com so we can help if you have any questions.

**Comment by smokku** [comment]
It's a bit worse at the moment.
https://choosealicense.com/no-permission/

> When you make a creative work (which includes code), the work is under exclusive copyright by default. Unless you include a license that specifies otherwise, nobody else can copy, distribute, or modify your work without being at risk of take-downs, shake-downs, or litigation. Once the work has other contributors (each a copyright holder), “nobody” starts including you.

**Comment by CodePoetSarah** [comment]
It also protects you as the originator of the project.

**Comment by sorgelig** [comment]
I'm sick from you guys. It's my hobby and and don't need such kind of suggestions. 
And no, i don't need/want any publishing on opensource.com !
Level 80 of procrastination.

**Comment by CodePoetSarah** [comment]
No problem,  I have canceled the article.  I am happy to keep the ticket
closed.  I just needed confirmation one way or another.

**Comment by smokku** [comment]
@sorgelig Please understand, thet the worries we have is that we _do_ care about this project and we want the best for it. There were nasty precedences when open source projects were forced to shot down, because of licensing issues.

@CodePoetSarah The issue here is probably because the [original MiST source code](https://github.com/mist-devel/mist-firmware) that MiSTer forked from is also without the license - unfortunately.

**Comment by CodePoetSarah** [comment]
Sorry, I misread your comment, I see you added a GPL,. Thanks
